galleries/examples/color/color_sequences.py

```diff
@@ -0,0 +1,61 @@
+"""
+=====================
+Named Color Sequences
+=====================
+
+Matplotlib's `~matplotlib.colors.ColorSequenceRegistry` allows access to
+predefined lists of colors by name e.g.
+``colors = matplotlib.color_sequences['Set1']``.  This example shows all of the
+built in color sequences.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import matplotlib as mpl
+
+
+def plot_color_sequences(names, ax):
+    # Display each named color sequence horizontally on the supplied axes.
+
+    for n, name in enumerate(names):
+        colors = mpl.color_sequences[name]
+        n_colors = len(colors)
+        x = np.arange(n_colors)
+        y = np.full_like(x, n)
+
+        ax.scatter(x, y, facecolor=colors, edgecolor='dimgray', s=200, zorder=2)
+
+        ax.set_yticks(range(len(names)), labels=names)
+        ax.grid(visible=True, axis='y')
+        ax.yaxis.set_inverted(True)
+        ax.xaxis.set_visible(False)
+
```

Comment:
How about making it even more minimal?

```suggestion
        ax.spines[:].set_visible(False)
        ax.tick_params(left=False)

```
![grafik](https://github.com/user-attachments/assets/a5728281-08d6-4ae3-b007-77dacbd8f52d)